src/BinSkim.Sdk/MetadataConditions.cs

```diff
@@ -22,6 +22,7 @@ public static class MetadataConditions
         public static readonly string ImageIsNot32BitBinary = SdkResources.MetadataCondition_ImageIsNot32BitBinary;
         public static readonly string ImageIsNot64BitBinary = SdkResources.MetadataCondition_ImageIsNot64BitBinary;
         public static readonly string ElfIsCoreNoneOrObject = SdkResources.MetadataCondition_ElfIsCoreNoneOrObject;
+        public static readonly string ImageIsArm64BitBinary = SdkResources.MetadataCondition_ImageIsARM64BitBinary;
```

Comment:
>MetadataCondition_ImageIsARM64BitBinary [](http://example.com/codeflow?start=75&length=39)

Casing of this is still wrong, s/be MetadataCondition_ImageIsArm64BitBinary. Sorry to nickle-and-dime you, generally we follow the .NET guidelines in all source code expressed stuff (like resx string names). Actual user-facing strings can retain capitalization exceptions if it's part of the public brand.

Note instances of 'Elf' and 'Wix' in this file, where a user-facing string might prefer 'ELF' or 'WIX'.